src/qml/InformationDrawer.qml

```diff
@@ -49,56 +48,165 @@ Item {
   Column {
     id: mainContent
     width: parent.width - 10
-    anchors.horizontalCenter: parent.horizontalCenter
+    leftPadding: 5
+    rightPadding: 5
     spacing: 8
 
-    NavigationInformationView {
-      id: navigationInformationView
+    Rectangle {
+      visible: navigationInformationViewEnabled
       width: parent.width
-      height: navigationInformationViewEnabled ? contentHeight : 0
+      height: childrenRect.height
+      color: Theme.mainBackgroundColorSemiOpaque
       radius: itemRadius
       clip: true
-      navigation: controller.navigation
+
+      Column {
```

Comment:
I really love the transparent and labled background. :)
also i think we might use a component to make this style and remove duplicated codes. i think its good to use `default property alias` in that component.

Reply:
@mohsenD98 , TIL about default property alias, thanks! Good point.